the_well/data/datasets.py

```diff
@@ -136,6 +136,9 @@ class WellDataset(Dataset):
             Whether to normalize data in the dataset
         normlization_type:
             What type of dataset normalization. Callable Options: ZSCORE and RMS
+        max_rollout_steps:
+            Maximum number of output steps to return in a single sample, mostly used for
+            full trajectory mode.
```

Comment:
```suggestion
            Maximum number of output steps to return in a single sample. Return the full trajectory if larger than its actual length.
```

Following @mikemccabe210 comment, it can be even more explicit that we return at most the full trajectory.